NotificationHubs/pom.xml

```diff
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.windowsazure</groupId>
 	<artifactId>Notification-Hubs-java-sdk</artifactId>
-	<version>0.0.5</version>
+	<version>0.0.6</version>
```

Comment:
This is a nit, but I think we could consider this a minor bump instead of patch since it's a non-breaking feature add. So `0.1.0` would be my suggested new version, if we want to follow semver.

Reply:
I don't like big numbers, but changes will be made.